adr/ADR-12.md

```diff
@@ -0,0 +1,133 @@
+# JWT library free jwt user generation
+
+|Metadata|Value|
+|--------|-----|
+|Date    |2021-07-20|
+|Author  | @mhanel, @aricart |
+|Status  |`Proposed`|
```

Comment:
Has to be one of Approved, Partially Implemented, Implemented, Rejected. In this case `Approved` is probably what you are aiming for once merged

Reply:
I figured I'll switch to the correct status once everyone reviewed and regenerate the readme as well.

Reply:
Actually, I'll set to Approved now and switch to Rejected in case this goes nowhere.